### PR TITLE
upd

### PR DESCRIPTION
**Pull request by jaxxstorm**
- start mapping outstanding tokens
- use auto mapping

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -128,67 +130,145 @@ func Provider() tfbridge.ProviderInfo {
...
+			"dynatrace_azure_credentials":             {Tok: dynatraceResource(mainMod, "AzureCredentials")},
+			"dynatrace_browser_monitor":               {Tok: dynatraceResource(mainMod, "BrowserMonitor")},
+			"dynatrace_browser_monitor_outage":        {Tok: dynatraceResource(mainMod, "BrowserMonitorOutage")},
+			"dynatrace_browser_monitor_performance":   {Tok: dynatraceResource(mainMod, "BrowserMonitorPerformance")},
+			"dynatrace_business_events_buckets":       {Tok: dynatraceResource(mainMod, "BusinessEventsBuckets")},
+			"dynatrace_business_events_metrics":       {Tok: dynatraceResource(mainMod, "BusinessEventsMetrics")},
+			"dynatrace_business_events_oneagent":      {Tok: dynatraceResource(mainMod, "BusinessEventsOneagent")},
+			"dynatrace_business_events_processing":    {Tok: dynatraceResource(mainMod, "BusinessEventsProcessing")},
+			"dynatrace_calculated_service_metric":     {Tok: dynatraceResource(mainMod, "CalculatedServiceMetric")},
+			"dynatrace_cloudfoundry_credentials":      {Tok: dynatraceResource(mainMod, "CloudfoundaryCredentials")},
```
Comment (williamsbdev) [inline provider/resources.go:160]:
```suggestion
			"dynatrace_cloudfoundry_credentials":      {Tok: dynatraceResource(mainMod, "CloudfoundryCredentials")},
```

Typo in the original mapping.

File: provider/resources.go [hunk 2]
```diff
@@ -128,67 +130,145 @@ func Provider() tfbridge.ProviderInfo {
...
+			"dynatrace_host_naming":                    {Tok: dynatraceResource(mainMod, "HostNaming")},
+			"dynatrace_http_monitor":                   {Tok: dynatraceResource(mainMod, "HttpMonitor")},
+			"dynatrace_ibm_mq_filters":                 {Tok: dynatraceResource(mainMod, "IbmMqFilters")},
+			"dynatrace_iam_group":                      {Tok: dynatraceResource(mainMod, "IamGroup")},
+			"dynatrace_iam_permission":                 {Tok: dynatraceResource(mainMod, "IamPermission")},
+			"dynatrace_iam_policy":                     {Tok: dynatraceResource(mainMod, "IamPolicy")},
+			"dynatrace_iam_policy_bindings":            {Tok: dynatraceResource(mainMod, "IamPolicyBindings")},
+			"dynatrace_iam_user":                       {Tok: dynatraceResource(mainMod, "IamUser")},
+			"dynatrace_issue_tracking":                 {Tok: dynatraceResource(mainMod, "IssueTracking")},
+			"dynatrace_ims_bridges":                    {Tok: dynatraceResource(mainMod, "ImsBridges")},
```
Comment (williamsbdev) [inline provider/resources.go:221]:
```suggestion
			"dynatrace_iam_group":                      {Tok: dynatraceResource(mainMod, "IamGroup")},
			"dynatrace_iam_permission":                 {Tok: dynatraceResource(mainMod, "IamPermission")},
			"dynatrace_iam_policy":                     {Tok: dynatraceResource(mainMod, "IamPolicy")},
			"dynatrace_iam_policy_bindings":            {Tok: dynatraceResource(mainMod, "IamPolicyBindings")},
			"dynatrace_iam_user":                       {Tok: dynatraceResource(mainMod, "IamUser")},
			"dynatrace_ibm_mq_filters":                 {Tok: dynatraceResource(mainMod, "IbmMqFilters")},
			"dynatrace_ims_bridges":                    {Tok: dynatraceResource(mainMod, "ImsBridges")},
			"dynatrace_issue_tracking":                 {Tok: dynatraceResource(mainMod, "IssueTracking")},
```

Comment (williamsbdev) [reply provider/resources.go:221]: Alphabetize the resources.

File: provider/resources.go [hunk 3]
```diff
@@ -128,67 +130,145 @@ func Provider() tfbridge.ProviderInfo {
...
+			"dynatrace_service_anomalies":        {Tok: dynatraceResource(mainMod, "ServiceAnomalies")},
+			"dynatrace_service_naming":           {Tok: dynatraceResource(mainMod, "ServiceNaming")},
+			"dynatrace_service_now_notification": {Tok: dynatraceResource(mainMod, "ServiceNowNotification")},
+			"dynatrace_slack_notification":       {Tok: dynatraceResource(mainMod, "SlackNotification")},
+			"dynatrace_slo":                      {Tok: dynatraceResource(mainMod, "Slo")},
+			"dynatrace_span_attribute":           {Tok: dynatraceResource(mainMod, "SpanAttributed")},
+			"dynatrace_span_capture_rule":        {Tok: dynatraceResource(mainMod, "SpanCaptureRule")},
+			"dynatrace_span_context_propagation": {Tok: dynatraceResource(mainMod, "SpanContextPropagation")},
+			"dynatrace_span_entry_point":         {Tok: dynatraceResource(mainMod, "SpanEntryPoint")},
+			"dynatrace_trello_notification":      {Tok: dynatraceResource(mainMod, "TrrelloNotification")},
```
Comment (williamsbdev) [inline provider/resources.go:265]:
```suggestion
			"dynatrace_trello_notification":      {Tok: dynatraceResource(mainMod, "TrelloNotification")},
```

Typo in the original mapping.

**Comment by williamsbdev** [comment]
@jaxxstorm - I noticed that `dotnet` build failed. I was looking at it and it appears that we're trying to build .NET 6?

https://github.com/lbrlabs/pulumi-dynatrace/actions/runs/5034861396/jobs/9030004078?pr=41#step:15:1259

It looks like our matrix currently accounts for .NET 3.1:

https://github.com/lbrlabs/pulumi-dynatrace/blob/main/.github/workflows/main.yml#L83-L85

**Comment by williamsbdev** [comment]
@jaxxstorm - I just saw that you updated the matrix with both .NET 3 and .NET 6. I'm not super familiar with .NET but when I was searching, it appears that we can have multiple versions of .NET installed at the same time. I think the problem with the build is that it needs both .NET 3 and .NET 6 installed at the same time and the matrix only installs one of the versions at a time.

**Comment by jaxxstorm** [comment]
@williamsbdev Thanks for the feedback. It seems this provider has a high priority for you, I am happy to transfer ownership if it's critical for your work

**Pull request merged**